spec/feature/add_space_spec.rb

```diff
@@ -0,0 +1,12 @@
+
+feature 'adding a space' do 
+  
+  scenario 'we can add a space' do 
+    visit('/add_space')
+    fill_in(:space_name, with: 'Bat Cave')
+    click_on 'Submit'
+    result = connection.exec('SELECT * FROM spaces')
```

Comment:
Should execution of this query be invoked here? I thought this is line should go when you visit 
``` 
get '/add_space' do
    erb :add_space
  end
```